src/views/ifsBrowser.js

```diff
@@ -498,11 +498,11 @@ module.exports = class ifsBrowserProvider {
                   message: `'${searchTerm}' in ${path}.`
                 });
 
-                const results = await Search.searchIFS(instance, path, searchTerm);
+                let results = await Search.searchIFS(instance, path, searchTerm);
 
                 if (results.length > 0) {
-
-                  setSearchResults(searchTerm, results);
+                  results = results.map(a => ({...a, label: a.path.replace(`${path}${path.endsWith(`/`) ? `` : `/`}`, ``)}));
```

Comment:
I am trying to understand what your `.replace` call is doing. 

Does this just get the basename of the path? Perhaps `path.basename` should be used instead?

Reply:
The `.replace` call is removing the search start path from the front of the absolute paths returned by `grep`, in order to show found files in subdirectories as relative to the search start path.

`basename` will not show the subdirectories, only the filenames - and what was used before (in the `HitSource` constructor).

Reply:
```suggestion
                  results = results.map(a => ({...a, label: a.path.replace(`${path}${path.endsWith(`/`) ? `` : `/`}`, ``)}));
```

Ok, perhaps want we should use here is `path.posix.relative` instead of basename?

```
> path.posix.relative('/home/JGORZINS/projects/manzan/', '/home/JGORZINS/projects/manzan/ile/src/pub_db2.sqlc')
'ile/src/pub_db2.sqlc'
```

Reply:
Jeez, I even looked at the path.posix functions, but missed out on the `path.posix.relative` function - fixed...